uniffi/src/interface/mod.rs

```diff
@@ -76,6 +76,7 @@ pub struct ComponentInterface {
     records: Vec<Record>,
     functions: Vec<Function>,
     objects: Vec<Object>,
+    error: Option<Enum>,
```

Comment:
Felt a little iffy about adding a full `error` piece here, but it seemed like a convenient spot if we end up going the "Generate `From<Error> for ExternError` route"